src/styles/shared/_typography.scss

```diff
@@ -13,34 +13,34 @@ $typography-condensed: em(640px);
 }
 
 @mixin text-style-caption {
```

Comment:
Leaving these `mixins` in for the scope of this PR. It would make sense to address them when we start thinking about composite components.